pensjon-brevbaker-api-model/src/main/kotlin/no/nav/pensjon/brev/api/model/maler/UngUfoerAuto.kt

```diff
@@ -10,13 +10,17 @@ data class UngUfoerAutoDto(
     val totaltUfoerePerMnd: Kroner,
     val ektefelle: InnvilgetTillegg?,
     val gjenlevende: InnvilgetTillegg?,
-    val fellesbarn: InnvilgetBarnetillegg?,
-    val saerkullsbarn: InnvilgetBarnetillegg?,
+    val fellesbarn: Barnetillegg?,
+    val saerkullsbarn: Barnetillegg?,
     val minsteytelseVedVirkSats: Double,
     val maanedligUfoeretrygdFoerSkatt: MaanedligUfoeretrygdFoerSkattDto?,
     val orienteringOmRettigheterUfoere: OrienteringOmRettigheterUfoereDto,
 ) {
     data class InnvilgetTillegg(val utbetalt: Boolean)
-    data class InnvilgetBarnetillegg(val utbetalt: Boolean, val antallBarn: Int, val inntektstak: Kroner)
-
+    data class Barnetillegg(
```

Comment:
Ser du har fjernet Innvilget fra navnet. Grunnen til at jeg inkluderte Innvilget i navnet var for å indikere at den kun skal ha en verdi om det er innvilget. Men det er kanskje åpenbart?

Reply:
Ja. I mitt hode leser jeg om er satt som om du har barnetillegg eller ikke.